src/overlays/actors/ovl_En_Sth/z_en_sth.c

```diff
@@ -61,104 +60,134 @@ static ColliderCylinderInit sCylinderInit = {
     { 30, 40, 0, { 0, 0, 0 } },
 };
 
-AnimationHeader* D_80B6D1C8[] = {
-    &ovl_En_Sth_Anim_003F50, &ovl_En_Sth_Anim_0045B4, &ovl_En_Sth_Anim_004CC0, &ovl_En_Sth_Anim_00533C,
-    &ovl_En_Sth_Anim_0059B8, &ovl_En_Sth_Anim_005E40, &ovl_En_Sth_Anim_006354, &ovl_En_Sth_Anim_00645C,
+typedef enum {
+    /* 0x0 */ STH_ANIM_SIGNALLING,   // default, waving arms at you from telescope, OOT: cured happy animation
+    /* 0x1 */ STH_ANIM_BENDING_DOWN, // default of second body
```

Comment:
Can you explain what you mean by "second body"? I presume it's when he uses Ahg instead of Sth, but I figure we should be clear here.

Reply:
Yes, using ahg body, yeah it should be more specific.